### ISSUE TITLE
CDM Technology Architecture Review WG - August 10th, 2023

### ISSUE THREAD

**Issue by eteridvalishvili** [finos/common-domain-model#2304]
# CDM Technology Architecture Working Group Minutes

## Date
August 10th, 2023 - 10 am ET / 2 pm BST

## Untracked attendees
- Fullname, Affiliation, (optional) GitHub username

## Meeting notices
- FINOS **Project leads** are responsible for observing the FINOS guidelines for [running project meetings](https://community.finos.org/docs/governance/meeting-procedures/). Project maintainers can find additional resources in the [FINOS Maintainers Cheatsheet](https://community.finos.org/docs/finos-maintainers-cheatsheet).

- **All participants** in FINOS project meetings are subject to the [LF Antitrust Policy](https://www.linuxfoundation.org/antitrust-policy/), the [FINOS Community Code of Conduct](https://community.finos.org/docs/governance/code-of-conduct) and all other [FINOS policies](https://community.finos.org/docs/governance/#policies). 

- FINOS meetings involve participation by industry competitors, and it is the intention of FINOS and the Linux Foundation to conduct all of its activities in accordance with applicable antitrust and competition laws. It is therefore extremely important that attendees adhere to meeting agendas, and be aware of, and not participate in, any activities that are prohibited under applicable US state, federal or foreign antitrust and competition laws. Please contact legal@finos.org with any questions.

- FINOS project meetings may be recorded for use solely by the FINOS team for administration purposes. In very limited instances, and with explicit approval, recordings may be made more widely available.

## Agenda
- [x] Convene & roll call (5mins)
- [x] Display [FINOS Antitrust Policy summary slide](https://community.finos.org/Compliance-Slides/Antitrust-Compliance-Slide.pdf) 
- [x] Review Meeting Notices (see above)
- [x] For Project Expo Demos reach out to [marketing@finos.org](mailto:marketing@finos.org) or [eteri.dvalishvili@finos.org](mailto:eteri.dvalishvili@finos.org) if you would like to showcase your use case on November 1st at the [OSFF NY 2023](https://www.finos.org/hosted-events/2023-11-01-open-source-in-finance-forum-new-york).
- [x] Approve past meeting minutes
  - [x] https://github.com/finos/common-domain-model/issues/2172
- [x] Update on Serialisation
  - [x] Refer to discussion #2180 
- [x] Issues to review for the next Meeting
  - [x] #2269 CDM Reference Data Lists
  - [x] #2015 Design Principles
- [x] CDM Versioning
  - [x] Presentation - [Semantic Versioning.pdf](https://github.com/finos/common-domain-model/files/12305616/Semantic.Versioning.pdf)
  - [x] Action Points
- [x] AOB, Q&A & Adjourn (5mins)

## Discussion points
**Versioning**
- Should make it as easy to migrate version as possible
- Linked to backwards compatibility etc.
- Seem to always be chasing versions at the moment
- Need support between versions for functional and structural changes
- Discussion not concluded, new github discussion to be formed for people to add feedback

**Serialisation**
- Overall the TAWG should come up with the recommendations as we are techs
- Then need to be able to describe the recommendations in plain English to the Steering WG (SWG)
- Separate group to be formed to work on this

**General**
- Recommendations need to be prioritised by the TAWG before passing to the SWG
- TAWG and SWG need to be able to get commitment from people to do the work
- TAWG can come up with use cases, don't have to rely on guidance from the SWG

## Action Items
- [x] Raise frequency of CDM Steering WG at the next 3TA CDM meeting - **Chris**
- [x] Raise new discussion about Versioning - **Chris** see #2305 
- [x] Create meeting to come up with Serialisation recommendations - **Dan**

### Zoom info
Join Zoom Meeting
https://zoom.us/j/99036410403?pwd=MXNINndYcEd0dTBUZERuYTFwRGpWZz09

Meeting ID: 990 3641 0403
Passcode: 745379
Find your local number: https://zoom.us/u/adtpQBjgT9

**Comment by chrisisla** [comment]
Chris Rayner / ISLA

**Comment by RonAtCFX** [comment]
Ron Kaim / CFX Consulting

**Comment by brianlynn2** [comment]
Brian Lynn / ISDA & Global Electronic Markets (GEM)

**Comment by manel-martos** [comment]
Manuel Martos / TradeHeader

**Comment by ahutusoru123** [comment]
Adrian Hutusoru / Fragmos Chain

**Comment by mgratacos** [comment]
Marc Gratacos / TradeHeader

**Comment by mmani99** [comment]
Mani Pillai / otcDigital

**Comment by jgavronsky** [comment]
Jane / FINOS

**Comment by eacunaISDA** [comment]
Eleonora / ISDA

**Comment by iansloyan** [comment]
Ian Sloyan / ISDA

**Comment by dschwartznyc** [comment]
Dan Schwartz / FT Advisory

**Comment by tomhealey-icma** [comment]
Tom Healey/ICMA
